notebooks/MosvizExample.ipynb

```diff
@@ -1,66 +1,157 @@
+    "warnings.simplefilter('ignore')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This imports MOSViz."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "mosviz = MosViz()\n",
-    "mosviz.app"
+    "from jdaviz.configs.mosviz.helper import MosViz as Mosviz"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The MOSViz parsers accepts lists of `Spectrum1D`, `SpectralCube`, and `CCDData` for 1D, 2D, and image data respectively. Alternatively, and the method used below, users can provide lists of file paths and MOSViz will internally attempt to parse them as their respective data type."
+    "But before we can use it, we need some data.\n",
+    "\n",
+    "The MOSViz parsers accept lists of `Spectrum1D`, `SpectralCube`, and `CCDData` for 1D, 2D, and image data, respectively. Alternatively, users can also provide lists of file paths and MOSViz will internally attempt to parse them as their respective data types."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": false
-   },
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run this cell if your desired data path is a temporary directory.\n",
+    "\n",
+    "import tempfile\n",
+    "data_dir = tempfile.gettempdir()"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "# *** OR ***\n",
+    "# Enable and run this cell if you have a different data path.\n",
+    "# Replace the value with your real path.\n",
+    "\n",
+    "data_dir = '/path/to/my_mosvis_data_dir'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run this cell if you wish to download and unzip our example data.\n",
+    "# This requires the \"zipfile\" package.\n",
+    "\n",
+    "from zipfile import ZipFile\n",
+    "\n",
+    "example_data = 'https://stsci.box.com/shared/static/ovyxi5eund92yoadvv01mynwt8t5n7jv.zip'\n",
+    "fn = download_file(example_data, cache=True)\n",
+    "with ZipFile(fn, 'r') as sample_data_zip:\n",
+    "    sample_data_zip.extractall(data_dir)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f'My data is in {data_dir}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have the data files extracted into `data_dir`, we can generate lists for MOSViz."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
    "outputs": [],
    "source": [
     "import glob\n",
+    "import pathlib\n",
     "\n",
     "spectra_1d = []\n",
     "spectra_2d = []\n",
     "images = []\n",
     "\n",
-    "from astropy.utils.data import download_file\n",
```

Comment:
This line is actually needed for the cell that downloads data.

Reply:
Whops... You caught me not downloading stuff! I added it back. Good catch!

Reply:
I fixed it. What about now?

Reply:
It does work, I just do not know if I like running 8 cells to see mosviz with data loaded. It's more organized but I don't feel like it's more efficient, if that makes sense.

Reply:
The currently order, you get MOSVis shown right off the bat, but by the time you get to the data loading, the viewer is scrolled out of view. Is that the experience you want user to have? I just thought it would be nice to have the viewer right there when you load the data.

Reply:
If you want me to move it back up, please reply again and let me know. 📎